packages/utils/index.ts

```diff
@@ -1,19 +1,18 @@
 import type { PackageManifest } from "query-registry";
 
 const githubUrlRegex =
-  /(?:git\+)?https?:\/\/github\.com\/([^\/]+\/[^\/]+)\.git/; // TODO: Don't trust this, it's chatgbd :)
+  /(?:git\+)?https?:\/\/github\.com\/([^\/]+)\/([^\/]+)\.git/; // TODO: Don't trust this, it's chatgbd :)
```

Comment:
what does this change do?

Reply:
Use two capture groups to capture the owner and repo so we don't need to `match[1].split("/")` later.

Reply:
amazing!